rocky/tests/conftest.py

```diff
@@ -26,6 +27,10 @@
 from octopoes.models.ooi.network import Network
 from rocky.scheduler import Task
 
+# Quiet faker locale messages down in tests.
+logger = logging.getLogger("faker")
+logger.setLevel(logging.INFO)
```

Comment:
Could be just a single statement without assignment, since you're not using this variable anymore
```suggestion
logging.getLogger("faker").setLevel(logging.INFO)
```